java/test/jmri/jmrit/logix/LinkedWarrantTest.java

```diff
@@ -32,7 +32,7 @@
     private WarrantManager _warrantMgr;
 
     // tests a warrant launching itself. (origin, destination the same to make continuous loop)
-    @Disabled("This test fails on CI")
+//    @Disabled("This test fails on CI")
```

Comment:
@petecressman 
Thanks for working on this. Unfortunately this class still have a lot of failures. See:
https://github.com/JMRI/JMRI/pull/10675#issuecomment-1034842030
https://github.com/JMRI/JMRI/pull/10675#issuecomment-1034970919